tests/integration/targets/dcnm_interface/tests/dcnm/dcnm_aa_fex_delete.yaml

```diff
@@ -9,19 +9,21 @@
   cisco.dcnm.dcnm_interface:
     check_deploy: "{{ test_data_common.check_deploy | bool }}"
     fabric: "{{ test_data_common.fabric }}"
-    state: overridden                     # only choose form [merged, replaced, deleted, overridden, query]
-  register: result
+    state: overridden  # only choose from [merged, replaced, deleted, overridden, query]
+  register: initial_override
```

Comment:
The variables in this file changed in some cases.  For instance, this line `result` changes to `initial_override`.  Are these fixes you made for this test?